src/ResourceManager/Relay/Commands.Relay/Models/AuthorizationRuleKeysAttributes.cs

```diff
@@ -46,7 +46,7 @@ public AuthorizationRuleKeysAttributes(string primaryConnectionString = default(
         /// Initializes a new instance of the AuthorizationRuleKeysAttributes class with a Azure.Management.Relay.Models.AuthorizationRuleKeys
         /// </summary>
         /// <param name="authorizationruleskeys"></param>
-        public AuthorizationRuleKeysAttributes(AuthorizationRuleKeys authorizationruleskeys)
+        public AuthorizationRuleKeysAttributes(AccessKeys authorizationruleskeys)
```

Comment:
just the name has been changed in the SDK, properties are same and cmdlet dosent expose it.